### ISSUE TITLE
ERROR: Cannot find Electron

### ISSUE THREAD

**Issue by mzaffalon** [JuliaPlots/PlotlyJS.jl#438]
I get this error after I have wiped out `.julia` and re-installed all packages for an unrelated reason.

The solution from #172 does not work anymore: 
```
julia> versioninfo()
Julia Version 1.8.0-rc1
Commit 6368fdc656 (2022-05-27 18:33 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 6 × Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, skylake)
  Threads: 1 on 6 virtual cores
Environment:
  JULIA_PKG_USE_CLI_GIT = true

(test) pkg> st
Status `C:\Users\michele.zaffalon\test\Project.toml`
  [ad839575] Blink v0.12.5
  [f0f68f2c] PlotlyJS v0.18.8
  [0f1e0344] WebIO v0.8.18

julia> using PlotlyJS, WebIO

julia> plot(scatter(y=randn(100)))
Error showing value of type PlotlyJS.SyncPlot:
ERROR: Cannot find Electron. Try `Blink.AtomShell.install()`.
<snip>

julia> using Blink

julia> Blink.AtomShell.remove()
ERROR: MethodError: no method matching remove()
Closest candidates are:
  remove(::Function, ::Lazy.List) at C:\Users\michele.zaffalon\.julia\packages\Lazy\9Xnd3\src\liblazy.jl:157
Stacktrace:
 [1] top-level scope
   @ REPL[4]:1

julia> Blink.AtomShell.install() # terminates Julia.
```

**Comment by mzaffalon** [comment]
This workaround works for me: https://github.com/JuliaGizmos/Blink.jl/issues/286#issuecomment-1221865085.

**Comment by mzaffalon** [comment]
Closed by #378.
